redaxo/src/core/lib/sql/sql.php

```diff
@@ -465,6 +468,33 @@ protected function isValueOf($feld, $prop)
         return strpos($this->getValue($feld), $prop) !== false;
     }
 
+    /**
+     * Adds a record for multi row/batch operations.
```

Comment:
You may invoke this method several times, before finally invoking save() or replace()